modules/transport/index.js

```diff
@@ -0,0 +1,27 @@
+import { API_ID } from '../API';
+
+import Transport from './Transport';
+
+/**
+ * Option for the default low level transport.
+ *
+ * @type {Object}
+ */
+const defaultTransportOptions = {};
+
+if (typeof API_ID === 'number') {
+    defaultTransportOptions.scope
+        = `jitsi_meet_external_api_${API_ID}`;
+}
+
+export const transport = new Transport({ defaultTransportOptions });
+
+/**
+ * Sets the transport to passed transport.
+ *
+ * @param {Object} newTransport - The new transport.
+ * @returns {void}
+ */
+window.useNewExternalTransport = function(newTransport) {
```

Comment:
IMHO it totally fits. Remote control is an API we provide, is it not? Whatever the API module meant in the past we can extend it now.